src/message/__tests__/__snapshots__/messageActions-test.js.snap

```diff
@@ -0,0 +1,59 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I went with snapshot tests because they make writing tests quicker. We don't need to write/maintain a set of outputs for our tests.